h/static/scripts/controllers/search-bar-controller.js

```diff
@@ -156,6 +188,16 @@ class SearchBarController extends Controller {
               querySelector('[data-ref="searchBarDropdownItemTitle"]').
               innerHTML.trim();
```

Comment:
handleClickOnItem has the same problem - I know this wasn't a change with your PR